test/src/main/java/org/apache/accumulo/test/ScanServerMultipleScansIT.java

```diff
@@ -0,0 +1,393 @@
+                break;
+              case 1:
+                scanner.setRange(new Range("row_0000000003", "row_0000000005"));
+                break;
+              case 2:
+                scanner.setRange(new Range("row_0000000006", "row_0000000008"));
+                break;
+              case 3:
+                scanner.setRange(new Range("row_0000000009"));
+                break;
+              default:
+                fail("Invalid threadNum");
+            }
+            scanner.setConsistencyLevel(ConsistencyLevel.EVENTUAL);
+
+            counter.addAndGet(Iterables.size(scanner));
+
+          } catch (TableNotFoundException e) {
+            fail("Table not found");
+          }
+        });
+
+        futures.add(future);
+      }
+      latch.countDown();
+      for (Future<?> future : futures) {
+        future.get();
+      }
+
+      assertEquals(100, counter.get());
+    }
+  }
+
+  @Test
+  public void testMutipleBatchScansSameTablet() throws Exception {
+    try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
+      String tableName = getUniqueNames(1)[0];
+
+      client.tableOperations().create(tableName);
+
+      ReadWriteIT.ingest(client, 10, 10, 50, 0, tableName);
+
+      client.tableOperations().flush(tableName, null, null, true);
+
+      final CountDownLatch latch = new CountDownLatch(1);
+
+      List<Future<?>> futures = new ArrayList<>(NUM_SCANS);
+
+      for (int i = 0; i < NUM_SCANS; i++) {
+        var future = executor.submit(() -> {
+          try {
+            latch.await();
+          } catch (InterruptedException e1) {
+            fail("InterruptedException waiting for latch");
+          }
+          try (BatchScanner scanner = client.createBatchScanner(tableName, Authorizations.EMPTY)) {
+            scanner.setRanges(Collections.singletonList(new Range()));
+            scanner.setConsistencyLevel(ConsistencyLevel.EVENTUAL);
+            assertEquals(100, Iterables.size(scanner));
+          } catch (TableNotFoundException e) {
+            fail("Table not found");
+          }
+        });
+        futures.add(future);
+      }
+      latch.countDown();
+      for (Future<?> future : futures) {
+        future.get();
+      }
+    }
+
+  }
+
+  @Test
+  public void testSingleBatchScanDifferentTablets() throws Exception {
+    try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
+      String tableName = getUniqueNames(1)[0];
+
+      client.tableOperations().create(tableName);
+      SortedSet<Text> splitPoints = new TreeSet<>();
+      splitPoints.add(new Text("row_0000000002\\0"));
+      splitPoints.add(new Text("row_0000000005\\0"));
+      splitPoints.add(new Text("row_0000000008\\0"));
+      client.tableOperations().addSplits(tableName, splitPoints);
+
+      @SuppressWarnings("deprecation")
+      Collection<Text> splits = client.tableOperations().getSplits(tableName);
```

Comment:
```suggestion
      Collection<Text> splits = client.tableOperations().listSplits(tableName);
```
Why not just use `listSplits()` for these instead?

Reply:
Updated to use `listSplits` in d416900f14ac0c0e88af37aff49cb3b97a6a8173. Good catch.